subsys/mgmt/mcumgr/smp/src/smp.c

```diff
@@ -232,11 +238,17 @@ static int smp_handle_single_payload(struct smp_streamer *cbuf, const struct smp
 end:
 #endif
 		/* End response payload. */
+#if defined(CONFIG_MCUMGR_MGMT_CUSTOM_PAYLOAD)
+		if (!handler->custom_payload) {
+#endif
 		if (!zcbor_map_end_encode(cbuf->writer->zs,
-					  CONFIG_MCUMGR_SMP_CBOR_MAX_MAIN_MAP_ENTRIES) &&
-		    rc == 0) {
+					CONFIG_MCUMGR_SMP_CBOR_MAX_MAIN_MAP_ENTRIES) &&
+			rc == 0) {
 			rc = MGMT_ERR_EMSGSIZE;
 		}
```

Comment:
Needs +1 tab indent

Reply:
Done :D

Reply:
Can you try and open the commit? this view here is marked "outdated". When I open the commit the indentation looks fine now

Reply:
What the code looks like:
```
#if defined(CONFIG_MCUMGR_MGMT_CUSTOM_PAYLOAD)
		if (!handler->custom_payload) {
#endif
		/* End response payload. */
		if (!zcbor_map_end_encode(cbuf->writer->zs,
					  CONFIG_MCUMGR_SMP_CBOR_MAX_MAIN_MAP_ENTRIES) &&
		    rc == 0) {
			rc = MGMT_ERR_EMSGSIZE;
		}
#if defined(CONFIG_MCUMGR_MGMT_CUSTOM_PAYLOAD)
		}
#endif
```
What it should look like:
```
#if defined(CONFIG_MCUMGR_MGMT_CUSTOM_PAYLOAD)
		if (!handler->custom_payload) {
#endif
			/* End response payload. */
			if (!zcbor_map_end_encode(cbuf->writer->zs,
						  CONFIG_MCUMGR_SMP_CBOR_MAX_MAIN_MAP_ENTRIES) &&
			    rc == 0) {
				rc = MGMT_ERR_EMSGSIZE;
			}
#if defined(CONFIG_MCUMGR_MGMT_CUSTOM_PAYLOAD)
		}
#endif
```

Reply:
Sorry, should be good now :)